src/Components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import './NewTodo.css';
+import cx from 'classnames';
+import { newTodoPropTypes } from '../PropTypes/PropTypes';
+
+class NewTodo extends React.Component {
+  state = {
+    users: [...this.props.users],
+    id: 3,
+    titleChange: '',
+    userChange: '',
+    errorTitle: null,
+    errorUser: null,
+  }
+
+  handleTitleChange = ({ target }) => {
+    this.setState({
+      titleChange: target.value,
+      errorTitle: null,
+    });
+  }
+
+  handleUserChange = ({ target }) => {
+    this.setState({
+      userChange: target.value,
+      errorUser: null,
+    });
+  }
+
+  handleButtonSubmit = (event) => {
+    event.preventDefault();
+    const {
+      titleChange, userChange, id, users,
+    } = this.state;
+    const { addNewTodo } = this.props;
+
+    if (!titleChange && !userChange) {
+      this.setState({
+        errorTitle: 'Write a title',
+        errorUser: 'Choose a user',
+      });
+    } else if (!titleChange) {
+      this.setState({
+        errorTitle: 'Write a title',
+      });
+    } else if (!userChange) {
+      this.setState({
+        errorUser: 'Choose a user',
+      });
+    } else {
+      addNewTodo({
+        userId: userChange.id,
+        id,
+        title: titleChange,
+        completed: false,
+        user: users
+          .find(user => user.name === userChange),
+      });
+      this.setState((prevState) => {
+        return ({
```

Comment:
rewrite returning object to avoid mutation

Reply:
rewrite names